[2022-09-27 06:08:58,690] {processor.py:163} INFO - Started process (PID=42) to work on /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-09-27 06:08:58,692] {processor.py:620} INFO - Processing file /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-09-27 06:08:58,694] {logging_mixin.py:109} INFO - [2022-09-27 06:08:58,694] {dagbag.py:500} INFO - Filling up the DagBag from /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-09-27 06:08:58,722] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-09-27 06:08:58,726] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:08:58,867] {logging_mixin.py:109} INFO - [2022-09-27 06:08:58,867] {dag.py:2389} INFO - Sync 1 DAGs
[2022-09-27 06:08:58,873] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:08:58,881] {logging_mixin.py:109} INFO - [2022-09-27 06:08:58,881] {dag.py:2921} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-09-27 06:08:58,917] {processor.py:171} INFO - Processing /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.232 seconds
[2022-09-27 06:09:38,795] {processor.py:163} INFO - Started process (PID=76) to work on /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-09-27 06:09:38,798] {processor.py:620} INFO - Processing file /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-09-27 06:09:38,800] {logging_mixin.py:109} INFO - [2022-09-27 06:09:38,800] {dagbag.py:500} INFO - Filling up the DagBag from /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-09-27 06:09:38,832] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-09-27 06:09:38,836] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:09:38,977] {logging_mixin.py:109} INFO - [2022-09-27 06:09:38,977] {dag.py:2389} INFO - Sync 1 DAGs
[2022-09-27 06:09:38,984] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:09:38,992] {logging_mixin.py:109} INFO - [2022-09-27 06:09:38,992] {dag.py:2921} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-09-27 06:09:39,024] {processor.py:171} INFO - Processing /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.233 seconds
[2022-09-27 06:10:18,464] {processor.py:163} INFO - Started process (PID=132) to work on /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-09-27 06:10:18,466] {processor.py:620} INFO - Processing file /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-09-27 06:10:18,468] {logging_mixin.py:109} INFO - [2022-09-27 06:10:18,468] {dagbag.py:500} INFO - Filling up the DagBag from /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-09-27 06:10:18,496] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-09-27 06:10:18,501] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:10:18,647] {logging_mixin.py:109} INFO - [2022-09-27 06:10:18,647] {dag.py:2389} INFO - Sync 1 DAGs
[2022-09-27 06:10:18,654] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:10:18,661] {logging_mixin.py:109} INFO - [2022-09-27 06:10:18,661] {dag.py:2921} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-09-27 06:10:18,696] {processor.py:171} INFO - Processing /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.237 seconds
[2022-09-27 06:10:53,635] {processor.py:163} INFO - Started process (PID=161) to work on /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-09-27 06:10:53,637] {processor.py:620} INFO - Processing file /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-09-27 06:10:53,639] {logging_mixin.py:109} INFO - [2022-09-27 06:10:53,639] {dagbag.py:500} INFO - Filling up the DagBag from /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-09-27 06:10:53,669] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-09-27 06:10:53,674] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:10:53,812] {logging_mixin.py:109} INFO - [2022-09-27 06:10:53,812] {dag.py:2389} INFO - Sync 1 DAGs
[2022-09-27 06:10:53,820] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:10:53,827] {logging_mixin.py:109} INFO - [2022-09-27 06:10:53,827] {dag.py:2921} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-09-27 06:10:53,859] {processor.py:171} INFO - Processing /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.229 seconds
[2022-09-27 06:11:27,409] {processor.py:163} INFO - Started process (PID=189) to work on /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-09-27 06:11:27,422] {processor.py:620} INFO - Processing file /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-09-27 06:11:27,445] {logging_mixin.py:109} INFO - [2022-09-27 06:11:27,444] {dagbag.py:500} INFO - Filling up the DagBag from /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-09-27 06:11:27,484] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-09-27 06:11:27,502] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:11:27,673] {logging_mixin.py:109} INFO - [2022-09-27 06:11:27,673] {dag.py:2389} INFO - Sync 1 DAGs
[2022-09-27 06:11:27,693] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:11:27,713] {logging_mixin.py:109} INFO - [2022-09-27 06:11:27,713] {dag.py:2921} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-09-27 06:11:27,775] {processor.py:171} INFO - Processing /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.369 seconds
[2022-09-27 06:11:59,151] {processor.py:163} INFO - Started process (PID=217) to work on /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-09-27 06:11:59,154] {processor.py:620} INFO - Processing file /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py for tasks to queue
[2022-09-27 06:11:59,156] {logging_mixin.py:109} INFO - [2022-09-27 06:11:59,156] {dagbag.py:500} INFO - Filling up the DagBag from /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-09-27 06:11:59,186] {processor.py:632} INFO - DAG(s) dict_keys(['tutorial_taskflow_api_etl']) retrieved from /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py
[2022-09-27 06:11:59,190] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/serialized_dag.py:120 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:11:59,328] {logging_mixin.py:109} INFO - [2022-09-27 06:11:59,328] {dag.py:2389} INFO - Sync 1 DAGs
[2022-09-27 06:11:59,334] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/dag.py:2426 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-09-27 06:11:59,341] {logging_mixin.py:109} INFO - [2022-09-27 06:11:59,341] {dag.py:2921} INFO - Setting next_dagrun for tutorial_taskflow_api_etl to None
[2022-09-27 06:11:59,373] {processor.py:171} INFO - Processing /usr/local/lib/python3.8/site-packages/airflow/example_dags/tutorial_taskflow_api_etl.py took 0.226 seconds
